examples/src/bin/uart.rs

```diff
@@ -8,22 +8,21 @@ use example_common::*;
 
 use cortex_m_rt::entry;
 use defmt::panic;
-use futures::pin_mut;
-use nrf52840_hal::gpio;
-
 use embassy::executor::{task, Executor};
-use embassy::io::{AsyncBufRead, AsyncBufReadExt, AsyncWrite, AsyncWriteExt};
+use embassy::time::{Duration, Timer};
 use embassy::util::Forever;
-use embassy_nrf::buffered_uarte;
-use embassy_nrf::interrupt;
+use embassy_nrf::{interrupt, pac, rtc, uarte};
+use futures::future::{select, Either};
+use nrf52840_hal::clocks;
+use nrf52840_hal::gpio;
 
 #[task]
 async fn run() {
-    let p = unwrap!(embassy_nrf::pac::Peripherals::take());
+    let p = unsafe { embassy_nrf::pac::Peripherals::steal() };
```

Comment:
why steal?

Reply:
Already taken in `main` to init RTC for the executor

Reply:
Ah, right. You can pass it as a param to `run()`. It's nice to minimize unsafe usage in examples

Reply:
That’s nifty! Didn’t know it is possible.